sparkle.js

```diff
@@ -22,15 +23,23 @@ window.onload = function() {
   resetState();
 
   function displayPixelatedImage() {
+    if(!canvas.width || !canvas.height) {
+      return;
+    }
```

Comment:
This function is called by `resetState` before image is loaded. I've added this sanity check to prevent error caused by calling `getImageData` on a canvas with `height === 0`.